DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -94,6 +94,9 @@ public void VisitIdentifier(DMASTIdentifier identifier)
                 case "args":
                     Result = new Expressions.Args(identifier.Location);
                     break;
+                case "opendream_procpath": // Our saner alternative to .....
```

Comment:
Can I hit you with `"_od_builtin_procpath"`? Might communicate that this is a compiler built-in a little better.

Reply:
Everyone can bikeshed it on Discord to their heart's content and then when the group reaches consensus I'll use whatever everyone agrees with.

Reply:
Considering the formatting of other OD stuff like `opendream_compiletimereadonly`, and the lack of any real interest from anyone else in arguing over the name, I'm kind of leaning towards just keeping it as-is tbh.